src/window.rs

```diff
@@ -380,6 +395,45 @@ impl Window {
     pub fn set_decorations(&self, decorations: bool) {
         self.window.set_decorations(decorations)
     }
+    
+    /// Centers the window on the current monitor the window lives in.
+    /// 
+    /// ## Platform-specific
+    /// 
+    /// This only works on Windows and X11.
+    /// it's not tested on macOS.
```

Comment:
You mean "It's not implemented on MacOS"? I don't see a MacOS impl anywhere.

At minimum, "it's" -> "It's"

Reply:
Nvm, I guess the default impls bellow act as MacOS's.

Anyways, "it's" -> "It's", "macOS" -> "MacOS"